dandiapi/api/views/asset.py

```diff
@@ -136,7 +136,7 @@ class Meta:
 
 
 class AssetViewSet(DetailSerializerMixin, GenericViewSet):
-    queryset = Asset.objects.all().order_by('created')
+    queryset = Asset.objects.all().select_related('zarr').order_by('created')
```

Comment:
Ignorant in django me wonders: 
- Since it seems to be a generic (not some zarr-specific) viewset, why only `zarr` to be select_related whenever e.g. in https://github.com/dandi/dandi-archive/pull/1216/files#diff-00bb976899730dcae9a407d74d487ec5ba609713007bb4bad93ca68da650cc1cR522 it is

```python
self.get_queryset().select_related('blob', 'embargoed_blob', 'zarr')
```
? e.g. I do see use of `.embargoed_blob` and `.blob` in the code below.
- do we really want *always* (since I guess it defines default behavior at this level) to do that (i.e. always add some minor overhead) or should it may be tuned per method depending on what it needs?

Reply:
While chatting with @waxlamp the answers are
- only `zarr` because some profiling on sentry showed that merging on that particular one was beneficial. Other ones might also be useful to be added, but nobody checked etc
- the answer is "may be" ;)